packages/elements/src/components/ui/icon/CircledIcon.tsx

```diff
@@ -45,3 +44,14 @@ const getBackgroundSize = (size: SizeVariant): string => {
       return exhaustSwitchCaseElseThrow(size);
   }
 };
+
+const getIconSize = (size: CircledIconSizeVariant): number => {
+  switch (size) {
+    case "small":
+      return 16;
+    case "medium":
+      return 20;
+    default:
+      return exhaustSwitchCaseElseThrow(size);
+  }
+};
```

Comment:
Why is background size a string, when this is a number? Use number everywhere!

Reply:
I'd argue the opposite.

Icon takes number and then converts it to rem (40 becomes 4.0rem).
Box can take a number, but just adds "px" suffix (40 becomes 40px).

Unit should be explicit so that it is clear where the UI should scale with accessible font size, and where it should not.

Regardless, that is not within the scope of notifications :) 
One could argue that the icon should not scale with accessible font size, but that would affect many many components. I'll take it up with Chris and Anton.